.travis.yml

```diff
@@ -1,43 +1,39 @@
 language: scala
 sudo: false
 
+jdk:
+- openjdk8
+
+before_install:
+- openssl aes-256-cbc -K $encrypted_316f730cabee_key -iv $encrypted_316f730cabee_iv -in ./project/secret -out ./project/zecret -d || true
+- git fetch --tags
+- export PATH=${PATH}:./vendor/bundle
+
 install:
 - rvm use 2.4.4 --install --fuzzy
 - gem update --system
 - gem install sass
-- gem install jekyll -v 3.8.2
+- gem install jekyll -v 3.8.3
 
 stages:
+- name: lint
```

Comment:
the less stages, the more parallelism
and we want parallelism :)

Reply:
yes but I also want a maintainable build definition. Let me mock around with this a bit, I'll see if I can figure this out..